lib/components/users.jsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019 balena.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Box,
+  Badge,
+  Container,
+  Flex,
+  Link,
+  Heading,
+  Txt,
+  Divider,
+} from 'rendition';
+
+export const name = 'Users';
+
+export const variants = metadata => {
+  const combinations = [];
+
+  if (metadata.data.github.usedBy) {
+    combinations.push({
+      users: metadata.data.github.usedBy,
+    });
+  }
+
+  return combinations;
+};
+
+const GITHUB_PROFILE_PATH = `https://github.com`;
+
+export const render = props => {
+  const list = props.users.map(({ repo, owner, description }) => {
+    return (
+      <>
```

Comment:
Is this a typo?

Reply:
No @jviotti 	This is a fragment https://reactjs.org/docs/fragments.html

Needed when you want to render two adjacent components without wrapping them with unnecessary dom elements